src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -83,125 +73,103 @@ Session replay minimizes impact on your app's performance with:
     
 Set a sampling rate (0%-100%) for the following samples:
 
-    * <DoNotTranslate>**Session sampling rate**</DoNotTranslate> records a random sample of all user sessions.
+    * <DoNotTranslate>**User sessions**</DoNotTranslate> records a random sample of all user sessions.
 
-    * <DoNotTranslate>**Errored session sample rate**</DoNotTranslate> records a random sample of all errored sessions. The agent buffers for up to 30 seconds on the client-side, and if errors occur, the session will be recorded. If an error does not occur, the replay is not harvested. 
+    * <DoNotTranslate>**Errored session**</DoNotTranslate> records a random sample of all errored sessions. The agent buffers for up to 30 seconds on the client-side, and if errors occur, the session will be recorded. If an error does not occur, the replay is not harvested. 
 
-For example, if you set the session sampling rate to 50% and the error sampling rate to 100%, it means that:
+Here's an example of configuring sampling rates: If you set the session sampling rate to 50% and the error sampling rate to 100%, it means that:
 
 * Half of all user sessions will be recorded.
 * All errored sessions will be recorded, even if they're not part of the 50% sample.
 
   </Step>
+<Step>
+### Configure privacy settings [#configure-privacy-settings]
```

Comment:
After reading this section, one thing that's not clear to me: what are New Relic's default privacy protections? Are there any? Is this a critical step to complete before implementing session replay?